rotkehlchen/tests/api/test_data_import.py

```diff
@@ -15,6 +15,9 @@
 from rotkehlchen.tests.utils.dataimport import (
     assert_cointracking_import_results,
     assert_cryptocom_import_results,
+    assert_blocfi_transactions_import_results,
+    assert_blocfi_trades_import_results,
```

Comment:
Change those 2 function names

```suggestion
    assert_blockfi_transactions_import_results,
    assert_blockfi_trades_import_results,
```